src/python/vehicle.py

```diff
@@ -0,0 +1,183 @@
+from typing import Dict
+import tensorflow as tf
+import numpy as np
+import random
+import json
+
+
+def generate_loss_function(pathToSpec:str, functionName:str, resources:Dict[str,any]):
+    '''
+    pathToSpec: path to the vehicle spec .vcl file
+    functionName: name of the function for which we want to create the loss function
+    resources: dictionary mapping from the name of the resources declared in the spec to the python implementation
+    '''
+
+    jsonDICT = generateJson(pathToSpec, functionName)
+    context = []
```

Comment:
Slightly pedantic, but I think the intent becomes slightly more clear if we called this `empty_context`? We can also declare it after we do the translation to json?